project-metrics/metrics_service/metrics/release_cherrypick_count.py

```diff
@@ -0,0 +1,42 @@
+"""Release Cherrypick Count metric."""
+
+import logging
+import sqlalchemy
+from typing import Text, Dict
+
+from database import db
+from database import models
+from metrics import base
+
+
+class ReleaseCherrypickCountMetric(base.Metric):
+  """A metric tracking the number of cherry-picks in releases."""
+
+  def _format_value(self, num_cherrypicks: float) -> Text:
+    return '%d PR%s/90d' % (num_cherrypicks, 's' if num_cherrypicks > 1 else '')
```

Comment:
Add parenthesis for clarity:
`(num_cherrypicks, ('s' if num_cherrypicks > 1 else ''))`

Nice trick!

Reply:
Done